src/InputChildren.tsx

```diff
@@ -1,20 +1,24 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { t, props } from 'tcomb-react';
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
 
-export const Props = {
-  children: t.maybe(t.ReactChildren),
-  wrapper: t.maybe(t.Object)
-};
+export namespace InputChildren {
+  export type Props = React.InputHTMLAttributes<HTMLInputElement> & {
+    children?: any,
```

Comment:
`any` children? Can't be more precise?

Reply:
can it? I thought TS didn't check this type anyway but only if it's undefined or not

Reply:
mmm, I think we used `React.ReactNode` or something similar everywhere in `react-components`.
Some sort of check should happen somehow (see Tablo for example)

Reply:
done